DCOS/regression-testing/src/script/disable_linux_agents_dcos_metrics.sh

```diff
@@ -26,7 +26,11 @@ declare -a cmds=(
   "sudo systemctl stop dcos-metrics-agent.service || echo skipped"
   "sudo systemctl stop dcos-metrics-agent.socket || echo skipped"
   "sudo systemctl disable dcos-metrics-agent.service || echo skipped"
-  "sudo systemctl disable dcos-metrics-agent.socket || echo skipped")
+  "sudo systemctl disable dcos-metrics-agent.socket || echo skipped"
+  "sudo systemctl stop dcos-checks-poststart.timer || echo skipped"
+  "sudo systemctl stop dcos-checks-poststart.service || echo skipped"
+  "sudo systemctl disable dcos-checks-poststart.timer || echo skipped"
+  "sudo systemctl disable dcos-checks-poststart.service || echo skipped")
```

Comment:
dcos-checks-poststart.service does lot of checking on the healthy state of a node, skipping it will make it more difficult to root cause issue

Reply:
This is how I ended up having healthy Linux agents as well. The `dcos-checks-poststart` queries metrics as well.

I agree with you that we should have a better approach to this: Disable metrics check for `dcos-check-poststart`. This is more of a workaround for now. We need to raise this into the next meeting with Mesosphere.

Reply:
Disabling is a costly workaround because it prevents us from checking the actual health of the Linux agent node.

A better approach is to remove the metrics service from the the Diagnostics service's check list by removing dcos-metrics-agent.service from dcos.target.wants directory. Then, the diagnostics will stop marking a agent node is not Unhealthy simply because metrics service was not running.

azureuser@dcos-lpubrs40129-24925853-000001:/etc/systemd/system/dcos.target.wants$ ls -l
t....
lrwxrwxrwx 1 root root 45 May  9 06:25 dcos-metrics-agent.socket -> /etc/systemd/system/dcos-metrics-agent.socket
...
azureuser@dcos-lpubrs40129-24925853-000001:/etc/systemd/system/dcos.target.wants$